lib/asset_host/asset.rb

```diff
@@ -103,9 +103,11 @@ def create(attributes={})
       def connection
         @connection ||= begin
           Faraday.new(
-            :url    => "http://#{config.server}",
-            :params => { auth_token: config.token }
+            :url    => "https://#{config.server}",
+            :params => { auth_token: config.token },
```

Comment:
Where does config.token come from? If it's a config value that we keep in our top secret key value store, then maybe we can change the above line to use a new config value too? Like `config.server_url` or something?